### PR TITLE
Case sensitive problem with `font-family`

### PR DESCRIPTION
**Pull request by evilebottnawi**
@davidtheclark ping

### CODE REVIEW

**Comment by evilebottnawi** [comment]
strange bug with github?

**Comment by davidtheclark** [comment]
Great.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # Head
 
 - Fixed: `block-closing-brace-newline-before` no longer delivers false positives for extra semicolon.
+- Fixed: `font-family-name-quotes` now handle case insensitive `font-family`.
```
Comment (davidtheclark) [inline CHANGELOG.md]: How about "is not case insensitive when hunting for `font-family` properties"

**Comment by davidtheclark** [comment]
Oops. I just added a comment about the changelog note. Feel free to just make that change and commit it directly to master.

**Comment by evilebottnawi** [comment]
@davidtheclark done

**Pull request merged**